spec/rspec_puppet_facts_spec.rb

```diff
@@ -265,6 +265,37 @@
       end
     end
 
+    context 'When testing AIX 7.1' do
+      subject {
+        on_supported_os(
+            {
+                :supported_os => [
+                    {
+                        "operatingsystem" => "AIX",
+                        "operatingsystemrelease" => [
+                            "7.1", "7100"
+                        ],
+                    },
+                ],
+                :facterversion => '3.9'
```

Comment:
I copied this concept from above (BSD tests), but isn't this cheating? I can think of how else this would work without submitting facts for every possible version of facter :-/